pos_invoice_confirm/static/src/js/screens.js

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2019 Druidoo - Iván Todorovich
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+odoo.define('pos_invoice_confirm.screens', function (require) {
+    "use strict";
+
+    var screens = require('point_of_sale.screens');
+    var PaymentScreenWidget = screens.PaymentScreenWidget;
+    var core = require('web.core');
+    var _t = core._t;
+
+    PaymentScreenWidget.include({
+        renderElement: function () {
+            var self = this;
+            this._super();
+            this.$('#button_next').click(function () {
```

Comment:
```suggestion
            this.$('.next').off('click').click(function() {
```
This is how you can replace the event callback, and avoid redefining the button